guides/common/modules/proc_securing-the-dhcp-api.adoc

```diff
@@ -6,7 +6,7 @@ By default, the dhcpd API listens to any host without access control.
 You can add an `omapi_key` to provide basic security.
 
 .Procedure
-. Install the required packages:
+. On your {SmartProxy}, install the required packages:
```

Comment:
I think this is confusing. We run also run a Smart Proxy on the `{ProjectServer}` and this section can apply both to `{ProjectServer}` or `{SmartProxyServer}`. We never say "on your `{SmartProxy}`.